services/wallet/api.go

```diff
@@ -317,7 +317,12 @@ func (api *API) GetEthereumChains(ctx context.Context, onlyEnabled bool) ([]*par
 
 func (api *API) FetchPrices(ctx context.Context, symbols []string, currencies []string) (map[string]map[string]float64, error) {
 	log.Debug("call to FetchPrices")
-	return fetchCryptoComparePrices(symbols, currencies)
```

Comment:
I tried to kind of build the `PriceManager` so that it would become a wrapper for Cryptocompare (in the future we might want to replace it or have other providers as backup). When we decide to do that, it'd be smart to move some of the things Cryptocompare does to the PriceManager, I just didn't think it was within the scope of this PR.